CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing Guidelines
+
+### Welcome to airbnb/javascript!
+
+First off, thank you for considering contributing to Airbnb JavaScript Style Guide. It's people like you that make airbnb/javascript such a great tool.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### Bug reports
+
+A bug is a demonstrable problem that is caused by the code in the reposityory. Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports: 
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the latest `master` or `next` branch in the repository.
+
+3. **Isolate the problem** &mdash; ideally create a reduced test case.
+
+A good bug report shouldn't leave others needing to chase you up for more information. Please try to be as detailed as possible in your report. What is your environment? What steps will reproduce the issue? What OS experiences the problem? What would you expect to be the outcome? All these details will help people to fix any potential bugs.
+
+Example: 
+
+
+> Short and descriptive example bug report title
+>
+> A summary of the issue and the browser/OS environment in which it occurs. If
+> suitable, include the steps required to reproduce the bug.
+>
+> 1. This is the first step
+> 2. This is the second step
+> 3. Further steps, etc.
+>
+> `<url>` - a link to the reduced test case
+>
+> Any other information you want to share that is relevant to the issue being
+> reported. This might include the lines of code that you have identified as
+> causing the bug, and potential solutions (and your opinions on their
+> merits).
+
+### Pull requests
+
+Good pull requests - patches, improvements, new features - are a fantastic
+help. They should remain focused in scope and avoid containing unrelated
+commits.
+
+**Please ask first** before embarking on any significant pull request (e.g.
+implementing features, refactoring code), otherwise you risk spending a lot of
+time working on something that the project's developers might not want to merge
+into the project.
+
+#### For new Contributors
+
+If you never created a pull request before, welcome 🎉 😄 [Here](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) is a great tutorial on how to send one :)
```

Comment:
videos aren't all that accessible, and this particular site/content is used as an enticement to sell access to further videos - can we find different content to link to?